sim/target-agnostic.mk

```diff
@@ -172,7 +172,8 @@ $(fpga_v): $(VERILOG) $(fpga_work_dir)/stamp
 	$(firesim_base_dir)/../scripts/repo_state_summary.sh > $(repo_state)
 	cp -f $< $@
 	sed -i "s/\$$random/64'b0/g" $@
-	sed -i 's/fatal/fatal(0, "")/g' $@
+	sed -i "s/void'(randomize(val));/val = '0;/g" $@
```

Comment:
Can you explain this?

Reply:
Ariane had a randomize statement that was getting through to synth, so I went ahead and just made it zero.

I actually don't think this is necessary anymore. I'll double check.